mech_client/interact.py

```diff
@@ -202,22 +212,47 @@ def send_request(  # pylint: disable=too-many-arguments
     :type tool: str
     :param price: The price for the request (default: 10_000_000_000_000_000).
     :type price: int
+    :param retries: Number of retries for sending a transaction
+    :type retries: int
+    :param timeout: Timeout to wait for the transaction
+    :type timeout: float
+    :param sleep: Amount of sleep before retrying the transaction
+    :type sleep: float
     """
     v1_file_hash_hex_truncated, v1_file_hash_hex = push_metadata_to_ipfs(prompt, tool)
     print(f"Prompt uploaded: https://gateway.autonolas.tech/ipfs/{v1_file_hash_hex}")
     method_name = "request"
     methord_args = {"data": v1_file_hash_hex_truncated}
     tx_args = {"sender_address": crypto.address, "value": price}
-    raw_transaction = ledger_api.build_transaction(
-        contract_instance=mech_contract,
-        method_name=method_name,
-        method_args=methord_args,
-        tx_args=tx_args,
-    )
-    raw_transaction["gas"] = 50_000
```

Comment:
Not required, https://github.com/valory-xyz/mech-client/pull/24/files#diff-64ddbc9531203f1e4503475a9f43904b2ad52498c3c4ec8c3c21fbab019117d9R68